daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -176,6 +176,54 @@ def _close(self):
         # TODO look into saving stats in /var/run maybe
         # self.key_manager.close()
 
+class RazerTartarusV2(_RippleKeyboard):
+    """
+    Class for Razer Tartarus V2
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Tartarus_V2(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x022B
+    DEDICATED_MACRO_KEYS = True
+    MATRIX_DIMS = [4, 6]
+
+    METHODS = ['get_device_type_keypad',
+               'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_key_row',
+               'set_reactive_effect',
+               'set_none_effect',
+               'set_custom_effect',
+               'set_wave_effect',
+               'set_static_effect',
+               'set_spectrum_effect',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'keypad_get_profile_led_red', 'keypad_set_profile_led_red',
+               'keypad_get_profile_led_green', 'keypad_set_profile_led_green',
+               'keypad_get_profile_led_blue', 'keypad_set_profile_led_blue',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_ripple_effect', 'set_ripple_effect_random_colour',
+               'get_game_mode', 'set_game_mode',
+               'keypad_get_mode_modifier', 'keypad_set_mode_modifier']
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1255/1255_tartarus_v2.png"
+
+    RAZER_URLS = {
+        "top_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h96/he9/8909630603294/razer-tartarus-v2-gallery-08.jpg",
+        "side_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h44/hea/8909630668830/razer-tartarus-v2-gallery-09.jpg",
+        "perspective_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/root/h56/hed/8909630734366/razer-tartarus-v2-gallery-10.jpg"
```

Comment:
If you could find any picture from the support.razer.com Domain, that would be better :) If not we can leave those

Reply:
Couldn't find anything.